Module 10 - Javascript and Ajax/classes/DeleteStudentController.cls

```diff
@@ -0,0 +1,29 @@
+public class DeleteStudentController
+{ 
+    ApexPages.StandardSetController setCon; 
+    public Boolean isNotEmpty {get; set;} 
+    public Boolean isEmpty {get; set;} 
+    public List<Id> listOfIds {get; set;} 
+    public Boolean toRenderStudentsName{get; set;} 
+    public DeleteStudentController(ApexPages.StandardSetController controller) 
+    { 
+        toRenderStudentsName = true; 
+        setCon = controller; 
+        listOfIds = new List<Id>(); 
+        for ( Student__c selectedStudent : (Student__c[])setCon.getSelected() ) 
+        { 
+            listOfIds.add(selectedStudent.Id);
+        } 
+        if(listOfIds.size() == 0) 
+        { 
+            isEmpty = true; 
+            toRenderStudentsName = false; 
+            final String msg = 'No record selected';
```

Comment:
we should not use aberrative.
if it's a constant means it's value will not change create a constant at class level.
final constants case should be in SCREAMING_SNAKE_CASE